src/main/java/TaskList.java

```diff
@@ -0,0 +1,112 @@
+import java.util.List;
+
+/**
+ * A class that handles any changes that occur within the
+ * list of tasks.
+ *
+ * @author Kai Chao
+ * @version 1.0
+ * @since 26-08-2020
+ */
+public class TaskList {
+
+    private List<Task> taskList;
+
+    TaskList(List<Task> tasks) {
+        this.taskList = tasks;
+    }
+
+    public int noOfTasks() {
```

Comment:
Hi, I think it is better to name this method as getNoOfTasks.

Reply:
Okay will do, thanks!